File: app/src/main/java/kyklab/quiet/ServiceKilledReceiver.java
package kyklab.quiet;

import android.content.BroadcastReceiver;
import android.content.Context;
import android.content.Intent;

public class ServiceKilledReceiver extends BroadcastReceiver {
    @Override
    public void onReceive(Context context, Intent intent) {
        VolumeWatcherService.startService(context);
    }
}

File: app/src/main/java/kyklab/quiet/ServiceKilledReceiver.kt
package kyklab.quiet

import android.content.BroadcastReceiver
import android.content.Context
import android.content.Intent

class ServiceKilledReceiver : BroadcastReceiver() {
    override fun onReceive(context: Context?, intent: Intent?) {
        context?.let { VolumeWatcherService.startService(it) }
    }
}